lib/Conch/Route/Validation.pm

```diff
@@ -33,12 +24,40 @@ sub routes {
     my $v = $r->any('/validation');
     $v->to({ controller => 'validation' });
 
+
+=head3 C<GET /validation>
+
+Response includes deactivated validations.
+
+=over 4
+
+=item * Requires Authentication
+
+=item * Response Schema: L<Validations|https://conch.joyent.us/schema/response/Validations>
+
+=back
+
+=cut
+
     # GET /validation
     $v->get('/')->to('#list');
 
     {
         my $with_validation = $v->under('/:validation_id_or_name')->to('#find_validation');
 
+
+=head3 C<GET /validation/:validation_id_or_name>
+
```

Comment:
Yes both can. I've however chosen to remove the copy for both of these for now because of the desire to work on the meaning and relationships between endpoints in future commits.

Since the endpoints are consistent with each other, I don't think removing the copy will be problematic for now.